src/widgets/StepperInputs/DurationEditor.js

```diff
@@ -0,0 +1,63 @@
+/* eslint-disable react/jsx-wrap-multilines */
+/* eslint-disable react/forbid-prop-types */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Text } from 'react-native';
+
+import { TextInputWithAffix } from '../TextInputs';
+import { Incrementor } from './Incrementor';
+import { DARKER_GREY } from '../../globalStyles';
+import { generalStrings } from '../../localization';
+
+const keepInRange = (num, min, max) => {
+  const temp = Math.min(num, max);
+  return Math.max(temp, min);
+};
```

Comment:
Wanted to abstract this as it's used in both components but.. I kinda didn't know where to put it.. and tbh it can just be a one liner in the component, it was just nicer to make a function name rather than a comment. 🤷